### PR TITLE
Allow users to request for multiple public ips with Azure Firewall

### PR DESCRIPTION
**Pull request by gimotwanMSFT**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by lirenhe** [comment]
@gimotwanMSFT, could you fix the prettiercheck error?  The BreakingChange errors could be ignored as you are working on a non-released version.

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/azureFirewall.json [hunk 1]
```diff
@@ -537,7 +551,6 @@
           "description": "The firewallPolicy associated with this azure firewall."
         },
         "hubIpAddresses": {
-          "readOnly": true,
```
Comment (lirenhe) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/azureFirewall.json:540]: why is 'readonly' removed?

Comment (majastrz) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/azureFirewall.json:540]: I would also like to know.

---
In reply to: [413481713](https://github.com/Azure/azure-rest-api-specs/pull/9151#discussion_r413481713) [](ancestors = 413481713)

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/examples/AzureFirewallPutInHub.json [hunk 2]
```diff
@@ -51,11 +51,14 @@
...
-                "address": "13.73.240.12"
-              }
-            ],
+            "publicIPs": {
+                "addresses": [
+                  {
+                    "address": "13.73.240.12"
+                  }
+                ],
+                "count": 1
```
Comment (majastrz) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/examples/AzureFirewallPutInHub.json:60]: >"count":  [](start = 16, length = 9)

Is the user expected to include the count on the PUT?

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/azureFirewall.json [hunk 3]
```diff
@@ -462,15 +462,29 @@
...
       "properties": {
-        "publicIPAddresses": {
+        "addresses": {
           "type": "array",
           "description": "List of Public IP addresses associated with azure firewall.",
           "items": {
             "$ref": "#/definitions/AzureFirewallPublicIPAddress"
           }
         },
+        "count": {
```
Comment (majastrz) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/azureFirewall.json:474]: >count [](start = 9, length = 5)

Is this intended to be a read-only or read-write property?

**Comment by number213** [comment]
Please rebase on https://github.com/Azure/azure-rest-api-specs/tree/network-may-release for next release cycle (`2020-05-01`)

**Pull request closed without merging**